hack/jenkins/e2e.sh

```diff
@@ -66,15 +66,45 @@ if [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
 fi
 
 # Specialized tests which should be skipped by default for projects.
-GCE_DEFAULT_SKIP_TEST_REGEX="Skipped|Density|Reboot|Restart|Example"
+GCE_DEFAULT_SKIP_TEST_REGEX="\
+Skipped|\
+Density|\
+Reboot|\
+Restart|\
+Example\
+"
 # The following tests are known to be flaky, and are thus run only in their own
 # -flaky- build variants.
-GCE_FLAKY_TEST_REGEX="Elasticsearch|Shell.*services|MaxPods.*"
+GCE_FLAKY_TEST_REGEX="\
+Elasticsearch\
+|Shell.*services\
+|MaxPods.*\
+"
 # Tests which are not able to be run in parallel.
-GCE_PARALLEL_SKIP_TEST_REGEX="${GCE_DEFAULT_SKIP_TEST_REGEX}|Etcd|NetworkingNew|Nodes\sNetwork|Nodes\sResize|MaxPods"
+GCE_PARALLEL_SKIP_TEST_REGEX="\
+${GCE_DEFAULT_SKIP_TEST_REGEX}\
+|Etcd\
+|NetworkingNew\
+|Nodes\sNetwork\
+|Nodes\sResize\
+|MaxPods\
+"
 # Tests which are known to be flaky when run in parallel.
 # TODO: figure out why GCE_FLAKY_TEST_REGEX is not a perfect subset of this list.
-GCE_PARALLEL_FLAKY_TEST_REGEX="Addon|Elasticsearch|Hostdir.*MOD|Networking.*intra|PD|ServiceAccounts|Service\sendpoints\slatency|Services.*change\sthe\stype|Services.*functioning\sexternal\sload\sbalancer|Services.*identically\snamed|Services.*release.*load\sbalancer|Shell|multiport\sendpoints"
+GCE_PARALLEL_FLAKY_TEST_REGEX="\
+Addon\
```

Comment:
You could removed this one, but since it's already merged we can do it in a separate PR (in fact it was my fault, because I forgot to remove it when I was promoting this test out of flaky).